README.md

```diff
@@ -119,7 +119,7 @@ To disable leader election, define an env variable named ENABLE_LEADER_ELECTION
 | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------: |
 | onDelete        | If it exists and has a delete value, delete the directory, if it exists and has a retain value, save the directory.                                                          | will be archived with name on the share: `archived-<volume.Name>` |
 | archiveOnDelete | If it exists and has a false value, delete the directory. if `onDelete` exists, `archiveOnDelete` will be ignored.                                                           | will be archived with name on the share: `archived-<volume.Name>` |
-| pathPattern     | Specifies a template for creating a directory path via PVC metadata's such as labels, annotations, name or namespace. To specify metadata use `${.PVC.<metadata>}`. Example: If folder should be named like `<pvc-namespace>-<pvc-name>`, use `${.PVC.namespace}-${.PVC.name}` as pathPattern. |                               n/a                                |
+| pathPattern     | Specifies a template for creating a directory path via PVC metadata's such as labels, annotations, name or namespace. To specify metadata use `${.PVC.<metadata>}`. Example: If folder should be named like `<pvc-namespace>-<pvc-name>`, use `${.PVC.namespace}-${.PVC.name}` as pathPattern. To use the default name use `${.PVC.namespace}-${.PVC.name}-${.PVC.volume}` as pathPattern. |                               n/a                                |
```

Comment:
i would say something more like `The default pathPattern is ...`.

and i am wondering maybe it should really be the default instead of using `strings.Join()`

https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/blob/498f518c64b95108ed0a2c13da9c267dc76381bc/cmd/nfs-subdir-external-provisioner/provisioner.go#L90 

:thinking:

Reply:
@yonatankahana do you want me to see if I can make this change? My goal was to make as minimal a change as possible.

Reply:
@robkooper sorry for the long time delay.
maybe for now it's ok, you can leave it for another PR, let's just have more clear documentation about the default value and we can get this merged :)

Reply:
@robkooper thank you, that would be great!